contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_match_xml_schema.py

```diff
@@ -165,7 +165,7 @@ def _prescriptive_renderer(
         include_column_name = (
             False if runtime_configuration.get("include_column_name") is False else True
         )
-        styling = runtime_configuration.get("styling")
+        runtime_configuration.get("styling")
```

Comment:
Not super familiar w/ rendering logic; do we need this bit at all?

Reply:
It's not being used so no.